public/exercism_completion.bash

```diff
@@ -1,7 +1,71 @@
-__from_exercism_config() {
-  COMPREPLY=()
-  local cur=${COMP_WORDS[COMP_CWORD]}
-  COMPREPLY=($(compgen -W 'demo debug configure fetch restore submit unsubmit tracks download help' -- $cur))
+_exercism () {
+  local cur prev
+
+  COMPREPLY=()   # Array variable storing the possible completions.
+  cur=${COMP_WORDS[COMP_CWORD]}
+  prev=${COMP_WORDS[COMP_CWORD-1]}
+
+  commands="configure debug download fetch list open
+  restore skip status submit tracks unsubmit
+  upgrade help"
+  tracks="csharp cpp clojure coffeescript lisp crystal
```

Comment:
A little command-line one liner for replacing the tracks with arbitrary text. It uses `perl`:
```bash
perl -i -pe 'BEGIN{undef $/;} s/(tracks=")(.*?)(")/$1NEW_TRACKS_HERE$3/smg' exercism_completion.bash
```

This is a possible solution, but I am not sure how it fits into your workflow.